api/all/src/main/java/io/opentelemetry/api/common/AttributeKey.java

```diff
@@ -24,7 +23,6 @@
   String getKey();
 
   /** Returns the type of attribute for this key. Useful for building switch statements. */
-  @Nonnull
```

Comment:
We don't default return values to Nonnull, do we? Is that even possible?

Reply:
Hmm. this is a good question, and I don't know the answer. We definitely *want* to have this be the default (non-null return values).